test/TestAssets/MSTestProject2/MSTestProject2.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-16"?>
```

Comment:
There is no difference.  Having 2 "identical" assets in different paths allows us to do parallel workflow tests. (I want to have a class that points to those projects in the end, rather than hardcoding names, and have some descriptions of them in the end (if we reuse them often.)).

Reply:
Oh ok makes sense.